cdap-app-templates/cdap-etl/cdap-data-streams-base/src/main/java/io/cdap/cdap/datastreams/DataStreamsPipelineSpec.java

```diff
@@ -34,27 +34,30 @@ public class DataStreamsPipelineSpec extends PipelineSpec {
   private final long batchIntervalMillis;
   private final String extraJavaOpts;
   private final boolean stopGracefully;
-  private final boolean checkpointsDisabled;
+  private final boolean checkpointsEnabled;
```

Comment:
The spec is used both internally, but I think it's also in some places by the UI (it gets serialized as one of the program properties). Can you check if the UI is using it anywhere?

Reply:
UI is using DataStreamsConfig object. According to the other comment, moved the state to a different class. Keeping the existing fields for backward compatibility, but marked them as deprecated.